src/sockaddr.rs

```diff
@@ -303,9 +303,11 @@ impl fmt::Debug for SockAddr {
             target_os = "ios",
             target_os = "macos",
             target_os = "netbsd",
+            target_os = "nto",
             target_os = "openbsd",
+            target_os = "tvos",
             target_os = "vxworks",
-            target_os = "nto",
+            target_os = "watchos",
```

Comment:
What happened with the removal of nto here?

Reply:
It moved up to line 306, I sorted the list.